cookbooks/hdfsdu/attributes/default.rb

```diff
@@ -0,0 +1,56 @@
+##################################
+#  hdfsdu specific attributes  #
+##################################
+
+default[:hdfsdu][:repo_url] = 'https://github.com/twitter/hdfs-du.git'
+default[:hdfsdu][:repo_branch] = 'master'
+default[:hdfsdu][:version] = '0.1.0'
+
+default[:hdfsdu][:download_dir] = '/tmp'
+default[:hdfsdu][:install_dir] = '/usr/lib'
+default[:hdfsdu][:service_dir] = '/etc/init'
+default[:hdfsdu][:log_dir] = '/var/log/hdfsdu'
+default[:hdfsdu][:data_dir] = '/var/lib/hdfsdu/data'
+default[:hdfsdu][:bin_dir] = "#{Dir.home}/chef-bcpc/bins"
+
+default[:hdfsdu][:service_download_url] = 'http://localhost' 
+default[:hdfsdu][:service_port] = 20000 
+default[:hdfsdu][:service_endpoint] = "http://localhost:#{node[:hdfsdu][:service_port]}/index.html" 
+default[:hdfsdu][:service_timeout] = 30 
+default[:hdfsdu][:default_db_port] = 14001
```

Comment:
Please add a comment that `default_db_port` is the port shipped by HDFSDU upstream and we replace it to be `db_port`.

Reply:
Updated